hphp/test/slow/reflection_classes/reflection_param_exception.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class A {
+  public static function bar($baz) {}
+}
+
+// Bad function.
+try {
+  new ReflectionParameter('A::foo', 'baz');
+} catch (ReflectionException $e) {
+  echo 'caught';
+}
+
+// Bad param name.
+try {
+  new ReflectionParameter('A::bar', 'not-baz');
```

Comment:
`not-baz` needs to be a non-string & non-int to trigger the 2nd newly added exception